model-driven-apps/custom-connectors/SecondaryRecordSource/solution/README.md

```diff
@@ -32,7 +32,7 @@ This sample includes a managed solution. This solution contains:
 - Configured table forms are "Account for Interactive Experiences" form (default for Customer Service Hub single-session) and the "Account for Multisession Experiences" form (default for Customer Service Workspace multi-session)
 - TimelineWallControl control is within first tab of these forms by default
 
-2. Download the managed solution "SecondaryRecordSource_SampleSolution_1_21111_1_managed.zip"
+2. Download the latest managed solution "SecondaryRecordSource_SampleSolution_1_22033_1_managed.zip"
```

Comment:
Do you need to specify latest if there is only one?

Reply:
fixed it